common/djangoapps/pipeline_mako/templates/i18n_helpers.html

```diff
@@ -0,0 +1,31 @@
+<%page expression_filter="h"/>
```

Comment:
Also, the `language_selector_config` def is mako-specific, as it reads the mako context and overrides the `LANGUAGE_SELECTOR` setting if it is present.